docker/prod/nginx.conf

```diff
@@ -74,7 +74,7 @@ http {
             fastcgi_param SYMFONY_ENV prod;
             fastcgi_param APP_ENV prod;
             fastcgi_param APP_DEBUG 0;
-            fastcgi_param HTTPS off;
+            fastcgi_param HTTPS on;
```

Comment:
This may introduce issues in production because the communication between the load-balancer and this app is not encrypted.